### PR TITLE
Meal and Cookbook classes

### PR DESCRIPTION
**Pull request by monwd**

### CODE REVIEW

**Comment by milkisbad** [comment]
approved

File: meal_planner_classes.py [hunk 1]
```diff
@@ -4,29 +4,53 @@
 import pandas as pd
 
 
-# Generate meal plan for one single day
-class SingleDayPlan:
+# # Generate meal plan for one single day
+class Meal:
+    def __init__(self, name, meal_type, category, ingredients):
+        # name of meal
+        self.name = str(name)
```
Comment (milkisbad) [inline meal_planner_classes.py:11]: why do you take `str(name)` ?

File: meal_planner_classes.py [hunk 2]
```diff
@@ -4,29 +4,53 @@
...
+
+class Cookbook:
+    def __init__(self, df_recipes):
+        self.df_recipes = df_recipes
 
     def draw_meal_type(self, meal_type):
-        meal = df_recipes.loc[df_recipes['Meal_type'] == meal_type].sample()
+        meal = self.df_recipes.apply(lambda row: row[df_recipes['Type'].isin([meal_type])])
         df_recipes.drop(meal.index[0], inplace=True)
         return meal.index[0]
```
Comment (milkisbad) [inline meal_planner_classes.py:30]: this method should return instance of class Meal

File: meal_planner_classes.py [hunk 3]
```diff
@@ -4,29 +4,53 @@
 import pandas as pd
 
 
-# Generate meal plan for one single day
-class SingleDayPlan:
+# # Generate meal plan for one single day
```
Comment (milkisbad) [inline meal_planner_classes.py:7]: useless comment?

File: meal_planner_classes.py [hunk 4]
```diff
@@ -4,29 +4,53 @@
...
+        self.ingredients = ingredients
+
+
+class Cookbook:
+    def __init__(self, df_recipes):
+        self.df_recipes = df_recipes
 
     def draw_meal_type(self, meal_type):
-        meal = df_recipes.loc[df_recipes['Meal_type'] == meal_type].sample()
+        meal = self.df_recipes.apply(lambda row: row[df_recipes['Type'].isin([meal_type])])
```
Comment (milkisbad) [inline meal_planner_classes.py:28]: `.apply` is bad way of doing it. The previous is better

File: meal_planner_classes.py (hunk 1 above)
Comment (monwd) [reply meal_planner_classes.py:11]: I was just testing something and forgot to remove. Removed for now.

**Pull request merged**